docs/rst/user_guide/atmosphere/homogeneous.rst

```diff
@@ -0,0 +1,64 @@
+.. _sec-atmosphere-homogeneous:
```

Comment:
Since this document demonstrates usage, I think it's best to make a tutorial out of it.